### PR TITLE
Add better error reporting/handling when importing SDR items

### LINKED ISSUE #271: Better error reporting/handling when importing items
**NOTE**: This ticket does not deal with handling unpublished items in a collection. See #198 for that.

When adding items to an exhibit from SDR, if you include unpublished druids in the list of items the process will fail (even for the published druids in the list, I believe) without any indication of the failure provided to the user.

Ideally, the process would:

- [x] Skip unpublished items but still add published items
- [x] Show an error message so the exhibit curator knows how many (and ideally, which) druids were not added

At the least, showing the error message would be helpful in letting the curator know to double-check the list of druids.

### PR DESCRIPTION
**Pull request by cbeer**
Fixes #271

screen shot 2016-08-01 at 2 05 57 pm

screen shot 2016-08-01 at 2 06 42 pm

### CODE REVIEW

File: app/views/dor_harvester/_form.html.erb [hunk 1]
```diff
@@ -1,10 +1,48 @@
...
-  <%= f.text_area :druid_list, rows: 10  %>
+<% harvester = DorHarvester.instance(current_exhibit) %>
+<div class="tab-content">
+  <div class="panel-group" id="sdr-accordion" role="tablist" aria-multiselectable="true">
+    <%= render partial: 'dor_harvester/status', locals: { harvester: harvester } %>
+    <div class="panel panel-default">
+      <div class="panel-heading" role="tab" id="sdr-item-submission">
+        <h4 class="panel-title">
+          <a role="button" data-toggle="collapse" data-parent="#sdr-accordion" href="#sdr-item-submission-section" aria-expanded="true" aria-controls="sdr-item-submission-section" class="">
+            <%= t ('admin.items.item-submission') %>
```
Comment (mjgiarlo) [inline app/views/dor_harvester/_form.html.erb]: Unnecessary space between `t` and open paren.

File: app/views/dor_harvester/_form.html.erb [hunk 2]
```diff
@@ -1,10 +1,48 @@
...
+            <span class="glyphicon glyphicon-chevron-right pull-right"></span>
+            <span class="glyphicon glyphicon-chevron-down pull-right"></span>
+          </a>
+        </h4>
+      </div>
+      <div id="sdr-item-submission-section" class="panel-collapse collapse in" role="tabpanel" aria-labelledby="sdr-item-submission" aria-expanded="true">
+        <div class="panel-body">
+          <div id="external_resource_tab_0" role="tabpanel" class="tab-pane active">
+            <p><%= t ('admin.items.item-submission-help_html') %></p>
```
Comment (mjgiarlo) [inline app/views/dor_harvester/_form.html.erb]: Unnecessary space between `t` and open paren.

File: app/views/dor_harvester/_status.html.erb [hunk 3]
```diff
@@ -0,0 +1,91 @@
+<div class="panel panel-default">
+  <div class="panel-heading" role="tab" id="sdr-item-status">
+    <h4 class="panel-title">
+      <a class="collapsed" role="button" data-toggle="collapse" data-parent="#sdr-accordion" href="#sdr-item-status-section" aria-expanded="false" aria-controls="sdr-item-status-section">
+        <%= t ('admin.items.item-status') %>
```
Comment (mjgiarlo) [inline app/views/dor_harvester/_status.html.erb]: More errant spaces between `t` and parens in this file as well.

File: config/locales/en.yml [hunk 4]
```diff
@@ -33,3 +34,13 @@ en:
...
+
+  admin:
+    items:
+      item-status: "Item status"
+      item-submission: "Item submission"
+      item-submission-help_html: "To add SDR items to the exhibit, enter collection or item druids below, one per line (e.g. qb438pg7646)."
+      object-druids: "Object druids"
+      object-druids-help_html: "Items below with the status of <em>published</em> were added to the exhibit. Items must be published before they can be added to the exhibit."
+      collection-druids: "Collection druids"
+      collection-druids-help_html: "Items associated with the collection druids below were added to the exhibit. If the number of items associated with a collection druid does not match your expectation, verify that all of its associated items have been published."
```
Comment (mjgiarlo) [inline config/locales/en.yml]: Worth clarifying what `its` means in `...verify that all of its associated items...`? @cbeer @ggeisler 

Perhaps `...verify that all of the collection's associated items...`?

Comment (cbeer) [inline config/locales/en.yml]: I'll defer to @ggeisler. I think it's ok enough as is, suspecting we'll probably make some more drastic changes soon enough.

**Pull request merged**